src/LaravelRequestDocsMiddleware.php

```diff
@@ -78,6 +79,11 @@ public function handle(Request $request, Closure $next): Response
             ],
         ];
 
+        if (Event::hasListeners(RequestHandled::class)) {
```

Comment:
This fix looks like a patch. 
The permanent fix is here: https://github.com/rakutentech/laravel-request-docs/pull/329